From 1444d26b4d6a6af780035b7d88640909c1ceb66e Mon Sep 17 00:00:00 2001
From: Andreas Mohr <and@gmx.li>
Date: Sun, 29 Nov 2020 11:28:47 +0000
Subject: (panel.c) fix shadow warning

Found by clang 11

panel.c:2960:15: error: declaration shadows a local variable [-Werror,-Wshadow]
        char *p;
              ^
panel.c:2939:13: note: previous declaration is here
    WPanel *p;
            ^

Signed-off-by: Andreas Mohr <and@gmx.li>
---
 src/filemanager/panel.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/filemanager/panel.c b/src/filemanager/panel.c
index 75a3fda20..c4e11a9d8 100644
--- a/src/filemanager/panel.c
+++ b/src/filemanager/panel.c
@@ -2936,7 +2936,7 @@ chdir_to_readlink (WPanel * panel)
     struct stat st;
     vfs_path_t *panel_fname_vpath;
     gboolean ok;
-    WPanel *p;
+    WPanel *cpanel;
 
     if (get_other_type () != view_listing)
         return;
@@ -2975,8 +2975,8 @@ chdir_to_readlink (WPanel * panel)
     else
         new_dir_vpath = vfs_path_append_new (panel->cwd_vpath, buffer, (char *) NULL);
 
-    p = change_panel ();
-    do_cd (p, new_dir_vpath, cd_exact);
+    cpanel = change_panel ();
+    do_cd (cpanel, new_dir_vpath, cd_exact);
     vfs_path_free (new_dir_vpath);
     (void) change_panel ();
 
